CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-## 5.3.4 (Next)
+## 5.4.1 (Next)
```

Comment:
This should be 5.4.0 and change the number in https://github.com/radar/distance_of_time_in_words/blob/master/lib/dotiw/version.rb

Reply:
I think it will be easier for you to handle the particulars of versioning rather than doing it through me, so I reverted this back to just stating "next".